postcss.config.js

```diff
@@ -0,0 +1,17 @@
+// @ts-nocheck
+module.exports = {
+	plugins: [
+		require("postcss-import"),
+		require("postcss-url"),
+		require("tailwindcss/nesting"),
+		require("tailwindcss")("./tailwind.config.js"),
+		require("postcss-combine-media-query"),
+		require("postcss-combine-duplicated-selectors")({
+			removeDuplicatedProperties: true,
+			removeDuplicatedValues: false,
+		}),
+		require("autoprefixer"),
+		require("cssnano")({ preset: "advanced" }),
+		require("postcss-reporter"),
```

Comment:
Could you let me know how these PostCSS plugins affect the outputted CSS compared to vanilla TailwindCSS w/autoprefixer? Curious to know if `postcss-combine-media-query` et al. help in reducing/optimizing the CSS file size.